kodilla-rps/src/main/java/com/kodilla/rps/RpsGame.java

```diff
@@ -0,0 +1,78 @@
+package com.kodilla.rps;
+
+public class RpsGame {
+    private RpsGameDefinition rpsGameDefinition;
+    private int roundNumber;
```

Comment:
jesli to jest numer aktualnej rundy, to nie moze to byc pole obiektu, ale zmienna lokalna metody play